.all-contributorsrc

```diff
@@ -449,6 +449,17 @@
             "avatar_url": [
                 "https://avatars.githubusercontent.com/u/12564882?v=4"
             ]
+        },
+        {
+            "login": "kimsin98",
```

Comment:
@kimsin98 you renamed yourself from @AKSoo? (it seems that tributors can't detect those yet)

I will assume so, and will just push that rename directly in this file.

Reply:
Yes, I did. Thank you!